src/Game.js

```diff
@@ -6,31 +6,36 @@ import data from '../src/data';
 
 
 class Game {
-  constructor() {
+  constructor(boards) {
     this.currentRound = {};
     this.roundTracker = 0;
```

Comment:
What’s the difference between `currentRound` and `roundTracker`? I’m imagining in my head that `currentRound` is an object that might contain the current categories/clues/etc. Could it also contain a key for the round number? Then you could get rid of `roundTracker`?